programs/clearing_house/src/controller/orders.rs

```diff
@@ -1164,6 +1165,7 @@ pub fn fulfill_order_with_amm(
     order_records: &mut Vec<OrderActionRecord>,
     override_base_asset_amount: Option<u128>,
     override_fill_price: Option<u128>, // todo probs dont need this since its the user_limit_price / current auction time
+    split_with_lps: bool,
```

Comment:
feel like you can already deduce this bool within this function if there exists an override_base_asset_amount/override_fill_price

(its only overwritten in jit)

Reply:
true - i think leaving it in is more explicit and on what we want the fcn to do